app/src/main/res/layout/activity_rating.xml

```diff
@@ -100,6 +100,7 @@
             android:layout_height="wrap_content"
             android:layout_marginStart="32dp"
             android:layout_marginTop="26dp"
+            android:inputType="text"
             android:layout_marginEnd="32dp"
```

Comment:
Also a warning there: Missing autoFillHints

Reply:
I see it as well! 👀